python/ql/lib/change-notes/2024-09-24-std-lib-models.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added several models of standard library functions and classes. This should make the analysis much more complete in cases where the standard library is not extracted.
```

Comment:
This change-note feels a bit vague, and to fully understand it you need to know that we're planning to not extract the standard library in the future. (which doesn't seem reasonable that all end-users will be aware of).

How about this instead?

```suggestion
* Added several models of standard library functions and classes, in anticipation of no longer extracting the standard library in a future release.
```